browser/brave_content_browser_client.cc

```diff
@@ -115,6 +117,40 @@ bool HandleURLRewrite(GURL* url,
   return false;
 }
 
+bool HandlePDFjsURLRewrite(GURL* url,
+                           content::BrowserContext* browser_context) {
+  if (url->SchemeIs(extensions::kExtensionScheme) &&
+      url->DomainIs(pdfjs_extension_id) && url->has_query()) {
+    return true;
+  }
+
+  return false;
+}
+
+bool HandlePDFjsURLReverseRewrite(GURL* url,
+                                  content::BrowserContext* browser_context) {
+  // Remove the PDF.js prefix from display URLs unless that's all there is.
+  if (url->SchemeIs(extensions::kExtensionScheme) &&
+      url->DomainIs(pdfjs_extension_id) && url->has_query()) {
+    // The path looks like this:
+    //   /content/web/viewer.html?file=https%3A%2F%2Fexample.com%2Ftest.pdf
+    base::StringPairs params;
+    if (base::SplitStringIntoKeyValuePairs(url->query(), '=', '&', &params)) {
+      for (size_t i = 0; i < params.size(); ++i) {
+        if (params[i].first == "file" && !params[i].second.empty()) {
+          net::UnescapeRule::Type rule =
+            net::UnescapeRule::URL_SPECIAL_CHARS_EXCEPT_PATH_SEPARATORS |
```

Comment:
nit: also continuation of line so should be indented 4 spaces from previous line